buildlib/fixup-include/netinet-in.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * This header is only used if sparse is detected and cmake arranges for it to
+ * replace all includes for glibc's <netinet/in.h>. Redefine the glibc socket
+ * data structures and macros in such a way that sparse can detect endianness
+ * mismatches when building with -D__CHECK_ENDIAN__.
+ */
+
+#ifndef _RDMA_NETINET_IN_H_
+#define _RDMA_NETINET_IN_H_
+
+#include <endian.h>
+#include_next <netinet/in.h>
+#include <util/compiler.h> /* __force */
+
+#define in_addr_t __be32
+
+#undef INADDR_ANY
+#define INADDR_ANY		htobe32(0)
+#undef INADDR_BROADCAST
+#define INADDR_BROADCAST	htobe32(0xffffffffU)
+#undef INADDR_NONE
+#define INADDR_NONE		htobe32(0xffffffffU)
+#undef INADDR_LOOPBACK
+#define INADDR_LOOPBACK		0x7f000001 /* Inet 127.0.0.1.  */
+
```

Comment:
This one is BE as well

I was also mulling over sed scripting /usr/include/net/in.h instead of providing a wrapper, it is hard to be complete with the wrapper approach..